src/index.html

```diff
@@ -1,14 +1,83 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div>
+        <a href="#">
+          <img src="./images/logo.png"
+            alt="Moyo logo"
+            class="logo"
+          >
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#Apple" class="nav-link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#Samsung"class="nav-link">
+               Samsung
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#Smartphones" class="nav-link">
+               Smartphones
+            </a>
+          </li>
+
+          <li class="nav-item"
+              data-qa="hover">
```

Comment:
you can write attributes from new line when you have more than two attributes
```suggestion
          <li class="nav-item" data-qa="hover">
```